Content/LeagueSandbox-Scripts/Maps/Map1.cs

```diff
@@ -257,10 +257,9 @@ public void Init(IMapScriptHandler map)
             map.ChangeTowerOnMapList("Turret_T1_C_07_A", TeamId.TEAM_BLUE, LaneID.MIDDLE, LaneID.BOTTOM);
 
             // Announcer events
-            map.AddAnnouncement(30 * 1000, Announces.WELCOME_TO_SR, true); // Welcome to SR
-            map.AddAnnouncement(FirstSpawnTime - 30 * 1000, Announces.THIRY_SECONDS_TO_MINIONS_SPAWN, true); // 30 seconds until minions spawn
-            map.AddAnnouncement(FirstSpawnTime, Announces.MINIONS_HAVE_SPAWNED, false); // Minions have spawned (90 * 1000)
-            map.AddAnnouncement(FirstSpawnTime, Announces.MINIONS_HAVE_SPAWNED2, false); // Minions have spawned [2] (90 * 1000)
+            map.AddAnnouncement(30 * 1000, 0x77, true); // Welcome to "Map"
```

Comment:
What do you mean? there's an EventID for it, it's `OnStartGameMessage1`

Reply:
I mean to use the enum rather than hardcoded values.
Since you are unable to use LeaguePackets as a reference, make a new EventID enum in GameServerCore and use that.

Reply:
Oh i see, would it be alright if i just copy and paste the whole file into GameServerCore then? Seems like it is what you're asking for

Reply:
Long as it builds and you setup proper casting when it is needed.